app/knowledge_panels.py

```diff
@@ -39,3 +44,43 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(facet: str, value: str):
+    if facet == "packaging":
+        plural_facet = facet
+        # That how the api read packaging
+    else:
+        plural_facet = p.plural(facet)
+    """
+    Changing other facet to plural because that how the search api works
+    """
```

Comment:
a facet_plural or plural method on Facet function would be cool (we may need it in other places)

Reply:
okay!